tests/kafkatest/tests/core/upgrade_test.py

```diff
@@ -145,3 +155,6 @@ def test_upgrade(self, from_kafka_version, to_message_format_version, compressio
         cluster_id = self.kafka.cluster_id()
         assert cluster_id is not None
         assert len(cluster_id) == 22
+
+        for node in self.kafka.nodes:
```

Comment:
nit: could probably push this loop into `check_protocol_errors`

Reply:
Fixed.